app/views/users/show.html.erb

```diff
@@ -1 +1,32 @@
-<h1>Show the current user</h1>
\ No newline at end of file
+
+<section class="show_user d-flex flex-column">
+    <div class="user-lists ">
+        <%= render "user_header", :user =>@user %>
+    </div>
+    <p class="ml-2">
+        lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+        lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+        lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+        lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+        lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+        lorem ipsum dolor sit amet,\adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+        lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+        lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+         dolor sit amet, consectetur adipisicing elit.
+        lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+
+    </p>
+
+    <div class=posts>
+    <% (1..3).each do |i|%>
+       <div class="post">
+            <h3>Post#<%= i %> </h3>
+            <p> 
+                 lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            </p>
+       </div>
+
```

Comment:
- [ ] Remember when you open a user page you should see only 3recent posts and when you want to see all user posts you will click on the link `see all posts`

Reply:
- [ ] This is the same to comments when a post have more than 5comments it should display only 5 of them